apps/researcher/src/app/[locale]/objects/[id]/useCurrentPublisher.ts

```diff
@@ -0,0 +1,16 @@
+import {create} from 'zustand';
+import {Organization} from '@/lib/api/definitions';
+
+export default create<Organization>(() => ({
```

Comment:
Thinking out load: is there a way to not create a copy of the 'Organization' data structure (which can change), but to re-use the existing one, with new defaults (empty strings)?